test.hxml

```diff
@@ -11,4 +11,4 @@
 --remap flash:openfl
 
 ## Neko
--cmd neko build/neko_test.n
\ No newline at end of file
+-neko build/neko_test.n
```

Comment:
Ah, that explains it. I'm running the tests by using `haxelib run munit test`, which only requires the `-neko` part to build (and run) the neko tests. When I run `haxe test.hxml`, it works as you've described.

I'm okay with either way (please let me know what you prefer so I can add it to the README).